src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
+
+	if( m_saveIntervalSlider->value() != value )
+	{
+		m_saveIntervalSlider->setValue( value );
+	}
+
+	m_saveIntervalLbl->setText( tr( "Auto save interval: %1 %2" ).arg(
```

Comment:
I haven't added two arguments like this to a translated string so I don't know how this looks to the translators. Should the space between the arguments be "hard coded" into the `minutes` string instead? This feels pretty solid though.